modules/cli/src/main/scala/scala/cli/commands/CompileOptions.scala

```diff
@@ -27,6 +35,13 @@ final case class CompileOptions(
   def buildOptions: BuildOptions =
     shared.buildOptions(enableJmh = false, jmhVersion = None)
 
+  def outputPath() =
+    output
+      .filter(_.nonEmpty)
+      .map { p =>
+        os.Path(Paths.get(p), os.pwd)
+      }
```

Comment:
This should do:
```suggestion
  def outputPath = output.filter(_.nonEmpty).map(p => os.Path(p, Os.pwd))
```
(`Os` being `scala.build.Os`)